### PR TITLE
[NOD-744] Wrap go routines with spawn

### PR DESCRIPTION
**Pull request by dereeno**

### CODE REVIEW

File: server/rpc/rpcserver.go [hunk 1]
```diff
@@ -623,12 +623,12 @@ func (s *Server) Start() {
 
 	for _, listener := range s.cfg.Listeners {
 		s.wg.Add(1)
-		go func(listener net.Listener) {
+		spawn(func() {
 			log.Infof("RPC server listening on %s", listener.Addr())
 			httpServer.Serve(listener)
 			log.Tracef("RPC listener done for %s", listener.Addr())
 			s.wg.Done()
-		}(listener)
```
Comment (someone235) [inline server/rpc/rpcserver.go]: This is actually needed

Comment (svarogg) [reply server/rpc/rpcserver.go]: @someone235 net.Listener is an interface, so no, it's not doing copying, so no problem.

File: connmgr/connmanager.go [hunk 2]
```diff
@@ -598,9 +598,12 @@ func (cm *ConnManager) Start() {
 	// provided a callback to be invoked when connections are accepted.
 	if cm.cfg.OnAccept != nil {
 		for _, listner := range cm.cfg.Listeners {
```
Comment (svarogg) [inline connmgr/connmanager.go]: Typo (not yours, but still please fix): listner -> listener

File: connmgr/connmanager.go [hunk 3]
```diff
@@ -594,8 +598,13 @@ func (cm *ConnManager) Start() {
...
+			// Declaring this variable is necessary so the anonymous function below will use it
+			// rather then the last element in the for loop.
+			var l = listner
```
Comment (svarogg) [inline connmgr/connmanager.go]: Mike L.:
> —> Without referring to the variable declaration convention, using "l" as a variable is not recommended and is disallowed by many conventions, in order not to confuse it with "1"

**Pull request merged**